src/rendering/renderers/shared/extract/ExtractSystem.ts

```diff
@@ -12,6 +12,12 @@ import type { GenerateTextureOptions } from './GenerateTextureSystem';
 
 type Formats = 'png' | 'jpg'; // Add other formats if needed
 
+const imageTypes: Record<Formats, string> = {
+    png: 'image/png',
+    jpg: 'image/jpeg',
+    // webp: 'image/webp',
```

Comment:
Can we enable webp? Was there a reason you disabled?

Also, the Formats type can be simplified to be `typeof keyof imageTypes` to be less redundant.

Reply:
type Formats = 'png' | 'jpg';
Because the current definition is two, we can add more if needed

Reply:
Go ahead and just add webp. It is supported with the browser API

Reply:
ok